src/commands/pod_install.yml

```diff
@@ -0,0 +1,16 @@
+description: install pods, use CircleCI cache to fetch CocoaPods specs
+
+# note: do not use the --project-directory pod param because many example
+# projects in the rn community repos rely on pod install to be run from ios directory
+# instead cd into the folder and then back
+
+parameters:
+  pod_install_directory:
+    type: string
+    description: The location of the "ios" directory
```

Comment:
Can this default to `"ios"` as that's by far the most likely location?

Reply:
done